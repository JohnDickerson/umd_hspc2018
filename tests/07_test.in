10
13/1
14/2
298619789/3211
298619789/92999
298619789/13
293089758/3299
293089758/88842
289564320/23456
12830723/3581
12830723/3583