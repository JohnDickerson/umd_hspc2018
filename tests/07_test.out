13
7
92999
3211
22970753
88842
3299
12345
3583
3581
